sources/scrapy_scraper_pipeline.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+import dlt
+
+from scrapy_scraper import scrapy_source
+from scrapy_scraper.helpers import Scraper
+from scrapy_scraper.settings import SOURCE_SCRAPY_QUEUE_SIZE
+from scrapy_scraper.spider import QuotesSpider
+from scrapy_scraper.types import BaseQueue
+
+
+def pipeline_runner(pipeline: dlt.Pipeline, queue: BaseQueue) -> None:
+    load_info = pipeline.run(
+        scrapy_source(queue=queue),
```

Comment:
Would it be possible to make `scrapy_source()` configurable via:
1. https://dlthub.com/docs/general-usage/credentials/configuration
2. function args
And also hide away the Queue from the end-user? (at least by default).

The idea of the verified sources is to make them reusable (but providing some sane configuration defaults). Let's try to think about the optimal user interface for `scrapy_source()`. 

1. First approach is to cover the case where the end user implements the spider. For example in this (https://quotes.toscrape.com) case I imagine we could pass the `start_urls` to `scrapy_source()` via config.toml (or an argument), and pass the Spider class as an agument. Like so:

      ```py
      from quotes_spider import QuotesSpider
      
      scrapy_source(
          start_urls=['https://quotes.toscrape.com'],
          spider=QuotesSpider
      )
      ```

2. Then, would it be possible to further simplify the configuration by implementing the spider implicitly? E.g. passing the `start_urls` as above, and also configuration that declaratively describes data extractors and pagination? (to cover the most common case).